configs/config.protectli_pt201

```diff
@@ -1,4 +1,29 @@
 CONFIG_LOCALVERSION="4.12"
 CONFIG_VENDOR_PROTECTLI=y
+CONFIG_CBFS_SIZE=0x200000
```

Comment:
Only 2 MB CBFS?

Reply:
I am pretty sure this was added by `savedefconfig`, what should the value be?